client/src/components/EventCard.tsx

```diff
@@ -4,6 +4,7 @@ import { isPast } from 'date-fns';
 import React from 'react';
 import { Chapter, Event } from '../generated/graphql';
 import { formatDate } from '../util/date';
+import ImagePlaceholder from './ImagePlaceholder';
```

Comment:
```suggestion
```

Reply:
Hi @Sboonny I've updated the code to reuse the existing placeholder.